toolchain/driver/BUILD

```diff
@@ -13,10 +15,66 @@ filegroup(
     data = glob(["testdata/**/*.carbon"]),
 )
 
+# Build rules to construct the install data tree for the Carbon toolchain.
+#
+# The directory `install` should mirror a Unix-y [FHS] style installation of
```

Comment:
Optional suggestion, but perhaps `root` or `base` instead of `install`, given you seem to be setting this up for use within `bazel` which doesn't strictly "install"?